packages/manager/modules/netapp/src/dashboard/index/translations/Messages_fr_FR.json

```diff
@@ -9,7 +9,13 @@
   "netapp_dashboard_network_configuration": "Configuration du réseau",
   "netapp_dashboard_network_configuration_status": "Statut",
   "netapp_dashboard_network_configuration_status_to_configure": "Prêt à configurer",
+  "netapp_dashboard_network_configuration_status_associating": "En cours de déploiement",
+  "netapp_dashboard_network_configuration_status_dissociating": "TODO",
```

Comment:
no french content ?

Reply:
I am waiting for it but I will add it in my last PR for this project